package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@onebeyond/notion-exporter",
   "author": "Yann Bolliger",
-  "contributors: ["One Beyond"],
+  "contributors": ["One Beyond"],
```

Comment:
I don't know how this passed previously 😨

Reply:
I accidentally introduced that on #2 as I made a code suggestion on the web and I didn't check that locally once I resolved the suggestion before merging the PR.